### PR TITLE
Added note about type:service + spotcast

### PR DESCRIPTION
**Pull request by exetico**
I find it pretty difficult to spot the type:service option, and get it working with my Google Setup. I sadly wish to have Sonos around - however, with 6 Google devices, i'm not going to swap them, anytime soon.

So for everyon with Google, i guess this could be informative for other people, too.

Thanks for a great solution, btw :-) I guess you're taking my 1. place for the best custom module - that's for sure!

### CODE REVIEW

**Comment by kalkih** [comment]
Hey, thanks for the PR, this could indeed be useful for other people.

File: README.md [hunk 1]
```diff
@@ -279,8 +279,17 @@ You can specify media shortcuts through the `shortcuts` option, either as a list
       - icon: mdi:dog
         type: script
         id: script.script_name
+      - name: Crooners
+        type: service
+        id: spotcast.start
+        data: {
+          "entity_id" : "media_player.googlehome1234",
+          "uri" : "spotify:playlist:37i9dQZF1DX9XiAcF7t1s5"
+        }
```
Comment (kalkih) [inline README.md]: I would prefer if we specified the data like this:
```yaml
data:
  entity_id: media_player.googlehome1234
  uri: spotify:playlist:37i9dQZF1DX9XiAcF7t1s5
```

File: README.md [hunk 2]
```diff
@@ -279,8 +279,17 @@ You can specify media shortcuts through the `shortcuts` option, either as a list
       - icon: mdi:dog
         type: script
         id: script.script_name
+      - name: Crooners
```
Comment (kalkih) [inline README.md]: Could we add a comment above this line? A short description like `#Trigger custom service` or similar, like the other shortcut examples.

File: README.md [hunk 3]
```diff
@@ -279,8 +279,17 @@ You can specify media shortcuts through the `shortcuts` option, either as a list
...
+        type: service
+        id: spotcast.start
+        data: {
+          "entity_id" : "media_player.googlehome1234",
+          "uri" : "spotify:playlist:37i9dQZF1DX9XiAcF7t1s5"
+        }
+
       ... # etc.
 ```
+_If you don't have Sonos, but want just a bit more control over playlists and so, a simple solution is to use the `type: service`-option, to trigger the`spotcast.start`-service. Remember to add the [required data](https://github.com/fondberg/spotcast#call-the-service), for spotcast to work. Also, kindly note that [spotcast](https://github.com/fondberg/spotcast) could be installed, for this to work. It's available in HACS.
```
Comment (kalkih) [inline README.md]: `could be installed` -> `should/needs/have to be installed`

File: README.md (hunk 1 above)
Comment (exetico) [reply README.md]: Ofc! I was to focused on the `data_template` options by editing this

File: README.md (hunk 3 above)
Comment (exetico) [reply README.md]: Sure. Type. Could <> Should. Let's go with `is required` :-)

**Comment by exetico** [comment]
Damn. Missed the `}`...

**Comment by exetico** [comment]
I guess that's all. Next time, i'll try and remember to use the preview-function before pushing changes ;-)

**Comment by kalkih** [comment]
Great! Thanks for the PR, merging now 🎉

**Pull request merged**